pubspec.yaml

```diff
@@ -8,7 +8,7 @@ environment:
   sdk: ^3.4.0
```

Comment:
I just realized a small mistake in v8.0.0; in order for the package:web version bracket to work, this `sdk` dependency should be: `sdk: ^3.3.0`.

Requiring `^3.4.0` here is (almost) the same as doing `web: ^1.0.0`.

I'll try to create a mini PR ASAP. Sorry for the churn, the new version is working great (and in Wasm!)